sbi/inference/posteriors/sbi_posterior.py

```diff
@@ -98,6 +105,7 @@ def log_prob(
 
         try:
             log_prob_fn = getattr(self, f"_log_prob_{self._alg_family}")
+            print("log_prob_fn", log_prob_fn)
```

Comment:
left over from debuggin? if not I would consider using logging or a more explicit print message.

Reply:
Yes, leftover, thanks.